onfido-java/src/main/models/document_classification.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "DocumentClassification",
+  "properties": {
```

Comment:
I haven't worked with Autofill much so you or Ciaran will know better, but do we return this in a `properties` bag? It doesn't look like it according to the API reference.

Reply:
This isn't a properties field, but rather the JSON Schema / OpenAPI type of properties field, where it lists the fields on the object

Reply:
Makes sense, thanks!